app/views/pages/developer-guide/apis/creating-items-from-external-api-call.liquid

```diff
@@ -2,28 +2,28 @@
 converter: markdown
 metadata:
   title: Creating Items from an External API Call
-  description: This guide will help you create items from an external API call. The example shows the steps of creating a Model from existing data through an API call to our service.
+  description: This guide will help you create items from an external API call. The example shows the steps of creating a Record from existing data through an API call to our service.
 ---
 
-This guide will help you create items from an external API call. The example shows the steps of creating a Model from existing data through an API call to our service.
+This guide will help you create items from an external API call. The example shows the steps of creating a Record from existing data through an API call to our service.
 
 ## Requirements
 This is an advanced tutorial. To follow it, you should be familiar with basic platformOS concepts, HTML, Liquid, APIs and Forms, and the topics in the Get Started section.
 
 * [platformOS Workflow](/developer-guide/platformos-workflow/development-workflow)
 * [Get Started](/get-started)
 * [APIs and Forms](/developer-guide/forms/form-configurations)
-* [Building a Contact Form with Models](/developer-guide/models/building-contact-form-with-model)
+* [Building a Contact Form with Records](/developer-guide/records/building-contact-form-with-records)
 
 ## Steps
 
 Creating items from an external API call is a four-step process:
 
 <div data-autosteps></div>
 
-### Step 1: Create Model Schema
+### Step 1: Create Table
 
-Create the following Model Schema:
+Create the following Table:
 
 ##### app/model_schemas/flowers.yml
```

Comment:
@diana-lakatos was this left out on purpose, for round 2?

Reply:
Yes, I only did the Developer Guide directory in round 1, and if that affected other files (e.g. links to topics, navigation), then I updated those as well.